engines/bastion_katello/test/repositories/new/new-repository.controller.test.js

```diff
@@ -2,10 +2,19 @@ describe('Controller: NewRepositoryController', function() {
     var $scope,
         FormUtils,
         GlobalNotification,
+        Setting,
         $httpBackend;
 
     beforeEach(module('Bastion.repositories', 'Bastion.test-mocks'));
 
+    beforeEach(module({
+        Setting: {
```

Comment:
You can do like the following instead of defining this full mock here -- https://github.com/Katello/katello/pull/5939/files#diff-9cef1387b3bc707bc14c38da782722e6R22